### ISSUE TITLE
Remember me doesn't work on login.html

### ISSUE THREAD

**Issue by krico** [krico/jas#602]
It seems like the "Remember me" checkbox has no effect and username is always remembered.

**Comment by pauljanes** [comment]
Yes today Jayne mentioned to me that even when she clicks Remember Me on her laptop it doesnt remember her next time she goes to the login page and she has to keep reentering her email and password each time.
